tests/test-frameworks/mixed/test

```diff
@@ -1,14 +1,10 @@
 #!/bin/bash -e
 . "$(dirname "$0")"/../../common.sh
 
-function test ()  {
-  out=$(bazel test ":mixed_$1")
-  echo "$out" | grep 'should pop values in last-in-first-out order'
-  echo "$out" | grep 'JUnit  many success'
-  echo "$out" | grep 'Dummy.bar: OK, proved property.'
-  bazel test ":mixed_$1" --test_arg=--verbosity=HIGH | grep 'org.scalacheck.ScalaCheckFramework: 1 tests'
-  bazel test ":mixed_$1" --test_arg=--verbosity=MEDIUM | grep -v 'org.scalacheck.ScalaCheckFramework: 1 tests'
-}
+out_2_13=$(bazel test :mixed_2_13)
+echo "$out_2_13" | grep 'have size 0'
+echo "$out_2_13" | grep 'JUnit many success'
 
-test "2_13"
-test "2_12"
+bazel test :mixed_2_12 | grep 'Dummy.bar: OK, proved property.'
+bazel test :mixed_2_12 --test_arg=--verbosity=HIGH | grep 'org.scalacheck.ScalaCheckFramework: 1 tests'
+bazel test :mixed_2_12 --test_arg=--verbosity=MEDIUM | grep -v 'org.scalacheck.ScalaCheckFramework: 1 tests'
```

Comment:
Because that's how this was originally. I made a mistake when I moved them all into the function. It looks like this https://github.com/lucidsoftware/rules_scala/blob/lucid-master/tests/test-frameworks/mixed/test on the master branch.